radio/src/targets/simu/simpgmspace.cpp

```diff
@@ -929,14 +929,24 @@ FRESULT f_mkdir (const TCHAR*)
   return FR_OK;
 }
 
-FRESULT f_unlink (const TCHAR*)
+FRESULT f_unlink (const TCHAR* name)
 {
+  char *path = convertSimuPath(name);
+  if (unlink(path)) {
```

Comment:
Ouch caution, I was used to simulate by removing real files, I will take care now ...